js/packery.js

```diff
@@ -429,6 +429,33 @@ Packery.prototype.itemDragEnd = function( elem ) {
   var onLayoutComplete = this._getDragEndLayoutComplete( elem, item );
 
   if ( item.needsPositioning ) {
+    // Check to see if the item being positioned is overlapping a stamped item.
+    for(var i = 0; i < this.stamps.length; i++) {
+      if(this.stamps[i] == item.element) {
+          // The currently moving item is stamped as well, so just ignore that one.
+          continue;
+      }
+      // Note: the stamped item MUST be an actual "item" so we can get the rect.x,rect.y coordinates.
+      // Otherwise the stamped item is just a DOM element.
+      var stamped = this.getItem(this.stamps[i]);
```

Comment:
You may want to look at using [`_getElementOffset`](https://github.com/metafizzy/outlayer/blob/v1.4.2/outlayer.js#L583) and [`_setRectSize`](https://github.com/metafizzy/packery/blob/v1.4.3/js/packery.js#L185), similar to how its used in [`_manageStamp`](https://github.com/metafizzy/packery/blob/v1.4.3/js/packery.js#L238)